certbot/tests/client_test.py

```diff
@@ -75,6 +76,8 @@ def test_email_retry(self, _rep, mock_get_email):
     @mock.patch("certbot.account.report_new_account")
     def test_email_invalid_noninteractive(self, _rep):
         from acme import messages
+        self.config.email = mock.MagicMock()
```

Comment:
Why do we set this to a mock object?

Reply:
Good question. This test was failing during my work and this mock was my way of returning this test to its previous defaults so I could understand its nature. This mock is actually impotent for this test, so I should remove it. Thanks for catching that.